stage3.0_first_views/setup.py

```diff
@@ -0,0 +1,23 @@
+from os.path import abspath, dirname, join
+from setuptools import setup, find_packages
+
+HERE = dirname(abspath(__file__))
+
+PKG_NAME = "pycasa"
+
+info = {}
+init_file = join(HERE, PKG_NAME, "__init__.py")
+
+with open(init_file) as fp:
+    exec(fp.read(), globals(), info)
+
+setup(
+    name=PKG_NAME,
+    version=info["__version__"],
+    description='Hello world for TraitsUI views',
```

Comment:
This is not really hello world anymore.

Reply:
True true. I added it because stage 3 would be their first brush with TraitsUI views. Is this better?
```suggestion
    description='Introduction to TraitsUI views',
```